RFC/src/RFC-0201_TariScript.md

```diff
@@ -394,84 +448,90 @@ validation steps.
-  record for the UTXO containing \\( C_a \\).
+* An input matching \\( C\_a \\), some output already on the blockchain.
 * The script input, \\( \input_a \\).
-* The height, \\( h \\), that the UTXO matching \\( C_a \\) was mined.
-* A valid signature, \\( (s_{Sa}\, R_{Sa}) \\) proving that she knows the private key, \\( k_{Sa} \\), corresponding to
-  \\( K_{Sa} \\), the public key left on the stack after executing \\( \script_a \\) with \\( \input_a \\).
-* An offset public key, \\( k_{Ob} \\).
-
-Since Bob will be countersigning the transaction, Alice can essentially construct a traditional MW output for Bob:
-
-She creates a new proto-output:
-* with the value _v_; Bob will provide the blinding factor (as per vanilla Mimblewimble),
-* her public nonce, \\( R_a \\) for the excess signature (also as per vanilla Mimblewimble),
-* and the hash of a `NO_OP` script (See [RFC 202](RFC-0202_TariScriptOpcodes.md)).
+* A valid script signature, \\( (a_{Sa}, b_{Sa}, R_{Sa}) \\) as per (3),(4) proving that she owns the commitment 
+  \\( C\_a \\), knows the private key, \\( k_{Sa} \\), corresponding to \\( K_{Sa} \\), the public key left on the stack 
+  after executing \\( \script_a \\) with \\( \input_a \\).
+* An _offset public key_, \\( k_{Ob} \\).
+* The _script offset_, \\( \so\\) with:
+$$
+\begin{aligned}
+\so  = k_{Sa} - k_{Ob}
+ \end{aligned}
+ \tag{10}
+$$
+* The sender signature \\( s_{Mb} \\) with: 
+$$
+\begin{aligned}
+  s_{Mb} = r_{mb} + k_{Ob} \hash{ \script_b \cat F_b \cat R_{Mb} }
+   \end{aligned}
+ \tag{11}
+$$
 
-Alice sends Bob this proto-UTXO, along with her signature nonce, as per the standard Mimblewimble protocol. However, she
-also provides Bob with \\( K_{Ob} \\), the script offset public key.
+Alice sends her excess, along with her signature nonce, as per the standard Mimblewimble protocol. However, she also 
+provides Bob with \\( \so \\), the _script offset_, as wel as a meta_signature \\(s_{Mb}\\) for his output \\(C_b\\).
 
 Bob can then complete his side of the transaction by completing the output:
 
 * Calculating the commitment, \\( C_b = k_b \cdot G + v \cdot H \\),
-* Choosing a private script key, \\( k_{Sb} \\),
-* Creating a range proof for \\( \hat{C}_b = (k_b + \rpc_b) \cdot G + v \cdot H \\), with
-
-  $$
-    \rpc_b = \hash{\script_b \cat F_b \cat K_{Ob} }
-  $$
 
 Bob then signs the kernel excess as usual:
 $$
-  s_b = r_b + k_b \hash{R_a + R_b \cat f \cat m }
+\begin{aligned}
+  s_b = r_b + k_b \hash{R_a + R_b \cat f \cat m } 
+  \end{aligned}
+ \tag{12}
 $$
 
-Bob returns the UTXO and partial signature along with his nonce, \\( R_b \\), back to Alice.
-
-Alice will then complete the transaction by calculating the script offset, \\( \so\\):
-$$
-\so  = k_{Sa} - k_{Ob} \HU_b
-$$
-
-She can then construct and broadcast the transaction to the network.
+Bob can then submit the completed transaction.
```

Comment:
Is this correct? Feels like a deviation from MW https://tlu.tarilabs.com/protocols/mimblewimble-1/MainReport.html